jpegenc/models/rgb2ycbcr/commons.py

```diff
@@ -0,0 +1,29 @@
+#!/bin/python
```

Comment:
and please watch the Champions League final if you are into this thing.
Just realized it is about to begin

2016-05-28 20:41 GMT+02:00 Nikolaos Kavvadias <nikolaos.kavvadias@gmail.com>
:

> Hi Merk,
>
> which myhdl are you using? master's HEAD from jandekaluwe or your fork ?
>
> 2016-05-28 20:40 GMT+02:00 Merkourios Katsimpris <notifications@github.com
> >:
>
>> In jpegenc/models/rgb2ycbcr/commons.py
>> <https://github.com/mkatsimpris/test_jpeg/pull/3#discussion_r64992442>:
>>
>> > @@ -0,0 +1,29 @@
>> > +#!/bin/python
>>
>> ok nikos I will change them all to true or false
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/mkatsimpris/test_jpeg/pull/3/files/0b87d64101aec6ad436384731ca94efb0ed53792#r64992442>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/AHgT_8VRPoKkNedH8tI9NhSYnHmzADg6ks5qGIwdgaJpZM4IlzwL>
>> .
>>
>
>